routes/index.js

```diff
@@ -21,4 +21,28 @@ const normalizeDate = (date) =>{
   return moment(date).format('MMMM Do YYYY')
 }
 
+router.post( '/create', ( req, res, next ) => {
+  const post = req.body
```

Comment:
Consider a different identifier for this variable - `post` is confusing since it already has a lot of meaning in HTTP.  Maybe `blogPost`, to make it explicit what you are referring to?